python/tvm/auto_scheduler/auto_schedule.py

```diff
@@ -117,14 +132,21 @@ def __init__(
         num_measure_trials=0,
         early_stopping=None,
         num_measures_per_round=64,
+        working_dir=None,
+        check_correctness=False,
         verbose=1,
         builder="local",
+        builder_n_parallel=-1,
```

Comment:
Seems we still omit here of `-1`, there is no place to have logic to handle `-1` now.

Reply:
This file is no longer used according to https://github.com/apache/tvm/pull/7053#issuecomment-741010499

Reply:
Ok.

Reply:
Yeah. I forget to delete it. Fixed by #7071